presto-product-tests/src/test/resources/sql-tests/testcases/aggregate/corr.sql

```diff
@@ -0,0 +1,2 @@
+-- database: presto; groups: mytest; tables: orders
+select round(corr(o_totalprice, o_orderkey),6) from orders
```

Comment:
nope, there was mild precision diff (after 8 or 9 decimal points) in results comparing psql baseline n postgres. So added a round.